packages/p2p/src/stores/my-ads-store.js

```diff
@@ -305,20 +309,23 @@ export default class MyAdsStore extends BaseStore {
             this.setIsTableLoading(true);
             this.setApiErrorMessage('');
         }
-
-        const { list_item_limit } = this.root_store.general_store;
-
+        const { floating_rate_store, general_store } = this.root_store;
         return new Promise(resolve => {
             requestWS({
                 p2p_advertiser_adverts: 1,
                 offset: startIndex,
-                limit: list_item_limit,
+                limit: general_store.list_item_limit,
             }).then(response => {
                 if (!response.error) {
                     const { list } = response.p2p_advertiser_adverts;
-                    this.setHasMoreItemsToLoad(list.length >= list_item_limit);
+                    this.setHasMoreItemsToLoad(list.length >= general_store.list_item_limit);
                     this.setAdverts(this.adverts.concat(list));
                     this.setMissingPaymentMethods(!!list.find(payment_method => !payment_method.payment_method_names));
+                    if (floating_rate_store.rate_type === ad_type.FLOAT) {
+                        floating_rate_store.setChangeAdAlert(checkForFixedRateAds(list));
+                    } else if (floating_rate_store.rate_type === ad_type.FIXED) {
+                        floating_rate_store.setChangeAdAlert(checkForFloatRateAds(list));
+                    }
```

Comment:
```suggestion
                    if (floating_rate_store.rate_type === ad_type.FLOAT) {
                        floating_rate_store.setChangeAdAlert(checkForFixedRateAds(list));
                    } else if (floating_rate_store.rate_type === ad_type.FIXED) {
                        floating_rate_store.setChangeAdAlert(checkForFloatRateAds(list));
                    }
```
So if you're checking for float ad_type you are firing functions for fixed? 🤯

Reply:
The logic is - if the country supports **float rate ads**, I need to check if there are any existing **Fixed rate ads** to show the warning banner. Also vice versa.
Hence if u see, I am checking if the rate_type is FLOAT, then I am invoking `checkForFixedRateAds`

I'll add comments to describe the functionality